plugins/woocommerce/src/Admin/Features/BlockEditorFeatureEnabled.php

```diff
@@ -50,6 +50,7 @@ public function enqueue_scripts() {
 		if ( ! empty( $post_type_object->template ) ) {
 			$editor_settings['template']     = $post_type_object->template;
 			$editor_settings['templateLock'] = ! empty( $post_type_object->template_lock ) ? $post_type_object->template_lock : false;
+			$editor_settings['__unstableResolvedAssets'] = gutenberg_resolve_assets_override();
```

Comment:
The `gutenberg_resolve_assets_override` is not available on a standalone WP version, and requires the Gutenberg plugin to be enabled in order to work.
We should probably wrap this in a `function_exists` and maybe create our own if this is required? Not sure if this would be added to WP 6.3 for example, but that be a relatively long wait.

Reply:
It seemed that we also rely on this function for some of the experimental components coming from  `@wordpress/block-editor`, or maybe they were also provided by the Gutenberg plugin.

Reply:
Copied this function over until it's available.  https://github.com/woocommerce/woocommerce/pull/37570/commits/55c9ccfb48865d340971d4a6784d813ea88fabb2